dependencies/pre-deployment-pvc-binding/pre-deployment-pvc-binding.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   name: test-pvc
   namespace: test-pvc-ns
 spec:
+  storageClassName: gp2
```

Comment:
The resources in this file are also deployed when using the flow in the readme. Users will not have this storage class available on their locally-deployed cluster.

Reply:
We can omit the storageClassName field and use the **default** StorageClass.
Same goes for the `PersistentVolume` configuration.